app/models/order.rb

```diff
@@ -38,6 +38,7 @@ def twilio_client
   end
 
   def send_initial_message
+    return unless ENV.key? 'TWILIO_ACCOUNT_SID'
```

Comment:
Should it throw some error or something so it doesn't keep creating the order? Or is it okay?